### PR TITLE
Fixed problem with broken response

### PR DESCRIPTION
**Pull request by antonpirker**

### CODE REVIEW

**Review by sl0thentr0py — APPROVED** [review]

**Comment by antonpirker** [comment]
This should now fix #1515

File: sentry_sdk/integrations/starlette.py [hunk 1]
```diff
@@ -339,7 +339,13 @@ async def form(self):
         curl -X POST http://localhost:8000/upload/somethign -H "Content-Type: application/x-www-form-urlencoded" -d "username=kevin&password=welcome123"
         curl -X POST http://localhost:8000/upload/somethign  -F username=Julian -F password=hello123
         """
-        return await self.request.form()
+        try:
```
Comment (sl0thentr0py) [inline sentry_sdk/integrations/starlette.py:342]: in general in lots of other places, we try to import stuff and fail there, but up to you however you prefer to do it
https://github.com/getsentry/sentry-python/blob/9e69fcf61a537ae30ea687f5b55957b6f3441935/sentry_sdk/integrations/flask.py#L23-L26

Comment (antonpirker) [reply sentry_sdk/integrations/starlette.py:341]: Ok. Done.

File: sentry_sdk/integrations/starlette.py [hunk 2]
```diff
@@ -339,6 +338,12 @@ async def form(self):
         curl -X POST http://localhost:8000/upload/somethign -H "Content-Type: application/x-www-form-urlencoded" -d "username=kevin&password=welcome123"
         curl -X POST http://localhost:8000/upload/somethign  -F username=Julian -F password=hello123
         """
+        try:
+            # Optional dependency of Starlette to parse form data.
+            import multipart  # type: ignore # noqa: F401
```
Comment (sl0thentr0py) [inline sentry_sdk/integrations/starlette.py:343]: I'd do at the top of the file
```python
try:
    import multipart
except ImportError:
    multipart = None
```

and then
```
if multipart is None:
    return None
```
in the function

Comment (antonpirker) [reply sentry_sdk/integrations/starlette.py]: nice!

File: sentry_sdk/integrations/starlette.py [hunk 3]
```diff
@@ -423,6 +423,7 @@ async def __call__(self, scope, receive, send):
         hub = Hub.current
         integration = hub.get_integration(StarletteIntegration)
         if integration is None:
+            await self.app(scope, receive, send)
```
Comment (axyjo) [inline sentry_sdk/integrations/starlette.py:434]: Thanks for this -- v1.8.0 is ~unusable with POST requests Starlette without this change

**Pull request merged**